ansible/roles/letsencrypt/tasks/main.yml

```diff
@@ -33,46 +18,32 @@
     - "/etc/nginx/sites-enabled"
     - "/var/www/letsencrypt"
 
-- name: check for /.well-known/acme-challenge
-  shell: "grep '/.well-known/acme-challenge' /etc/nginx/sites-enabled/*"
-  changed_when: false
-  ignore_errors: yes
-  register: acme_challenge_present
-
-- name: Add letsencrypt vhost
+- name: Add letsencrypt dedicated sites-enabled file
   template:
     src: templates/letsencrypt-http
     dest: /etc/nginx/sites-enabled/letsencrypt-http
+    mode: 0644
+    owner: root
```

Comment:
The pattern we were using up until now was that of writing directly inside of the default nginx config by calling the template functions: https://github.com/ooni/sysadmin/blob/master/ansible/templates/common.j2.

The reason to do so is that `letsencrypt-http` will be mapped to port 80, meaning that we are not able to serve the website on port 80 or do a redirect to port 443 in cases in which that is needed: https://github.com/ooni/sysadmin/blob/8e9c599b5f845a2914ed0c52c9191e5f40a17eb6/ansible/roles/letsencrypt/templates/letsencrypt-http